src/scriptworker/version.py

```diff
@@ -54,7 +54,7 @@ def get_version_string(version: Union[ShortVerType, LongVerType]) -> str:
 
 # 1}}}
 # Semantic versioning 2.0.0  http://semver.org/
-__version__ = (50, 0, 0)
+__version__ = (50, 1, 0)
```

Comment:
Nit: Let's extract the version bump in a separate commit. If I'm not mistaken, it's the trend in this repo.